bin/git-scp

```diff
@@ -107,6 +107,7 @@ function scp_and_stage
 	if [ -n "$list" ]
 	then
 		local _TMP=${0///}
+        # shellcheck disable=SC2086
```

Comment:
Bad indent and bad location?

Reply:
Hey, tabs vs spaces thing. Sorry I rebased the branch to main instead of merge from old habits :(